liubai-frontends/liubai-weixin/miniprogram/utils/LiuTime.ts

```
@@ -0,0 +1,37 @@
+    return new Date().getTime() + this.diff
+  }
+
+  static getTimezone() {
+    const d = new Date()
+    const t = d.getTimezoneOffset()
+    const t2 = -t / 60
+    return t2
+  }
+
+  static setDiff(val: number) {
+    this.diff = val
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)